homebrew_releaser/formula.py

```diff
@@ -50,6 +51,13 @@ def generate_formula_data(
         if first_word_of_desc[0].lower() in articles:
             description = first_word_of_desc[1].strip().capitalize()
 
+        depends_on_content = ''
+
+        if depends_on:
+            for dependency in depends_on.strip().split('\n'):
```

Comment:
You'll definitely want to put this back to `dependencies = [dependency.strip() for dependency in depends_on.split('\n') if dependency]` per the convo on the PR elsewhere.

Reply:
This is done, thank you :)